contracts/libraries/MultiSend.sol

```diff
@@ -64,8 +64,10 @@ contract MultiSend {
                 }
                 if eq(success, 0) {
                     let errorLength := returndatasize()
-                    returndatacopy(0, 0, errorLength)
-                    revert(0, errorLength)
+                    let returnPtr := mload(0x40)
+                    mstore(0x40, add(returnPtr, returndatasize()))
```

Comment:
`mstore` is not needed for memory safety - see https://github.com/safe-global/safe-contracts/issues/544#issuecomment-1835746635

Reply:
Will be removed 👍🏾